infra/src/main/resources/i18n/translations.fi.json

```diff
@@ -89,7 +89,8 @@
         "entity-not-found-on-path": "Tietoa ei löytynyt: {{path}}",
         "request-failed": "Pyyntö epäonnistui: {{path}}",
         "copied-to-clipboard": "Kopioitu leikepöydälle",
-        "copy-details-to-clipboard": "Kopioi tiedot leikepöydälle"
+        "copy-details-to-clipboard": "Kopioi tiedot leikepöydälle",
+        "expected-non-null": "Pyyntö palautti null-arvon"
```

Comment:
Päädyin lopulta tiskin ohi tällaiseen virheilmoitukseen non-nullable-nulleille

Reply:
Melko tekninen viesti käyttäjälle, mutta aika hankalaa tilannetta olisi tulkita riippumatta virheviestin sisällöstä. Toisaalta tässä käyttäjä ei välttämättä tiedä, että onko "null" arvo virheellinen vai haluttu (ja voi jäädä ihmettelemään miksi jotakin ei näytetä/miksei jotain tapahtunut).

Mun puolesta kuitenkin 🆗 näin.

Reply:
Joo "tyhjä" voisi olla parempi termi suomeksi